projects/uitdatabank/docs/terms-of-use/terms-of-use.md

```diff
@@ -0,0 +1,107 @@
+# Gebruiksvoorwaarden publicatietools
+
+(dd 20/06/2023)
+
+Onder publicatietools wordt begrepen:
+
+* UiTdatabank API
+* Agendawidgets: agendamodule gehost door publiq
+* Culturefeed Module Suite: een set van agenda-modules gebaseerd op Drupal
+
+De agendawidgets en Culturefeed Modules zijn gebouwd bovenop de UiTdatabank API.
```

Comment:
(Same comment for line 58): While the Culturefeed Modules are being upgraded by a third party, instead of deprecated as initially planned. There is no mention of Culturefeed in the rest of the docs, so I wonder if it belongs here?(Open question, legally it might be wise to put it here).

Reply:
Yes, the terms of use are definitely outdated. I already wrote a new version (without mentioning culturefeed modules, with information about auth0 being a processor of user data, etc.) but it needs a review of a jurist. So for now I just copied the terms of use available on [documentatie.uitdatabank.be](https://documentatie.uitdatabank.be/content/integratie-op-je-eigen-website/latest/gebruiksvoorwaarden.html). To be continued!